.github/workflows/build.yml

```diff
@@ -119,6 +124,11 @@ jobs:
         id: nb_workers
         run: python -c 'import os; print(f"nb_workers={max(1,os.cpu_count()-1)}")' >> $GITHUB_OUTPUT
 
+      - name: Build docs
```

Comment:
indeed it's going to be long since there's a lot of preparation (like environment installation).

You can just add `:python-client:sphinx-build` to the Build step, in this case gradle will build a DAG of tasks and will share some of the common steps